content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -142,6 +156,9 @@ using Datadog.Trace;
 using NLog;
 
 // there must be spans started and active before this block.
+using (MappedDiagnosticsLogicalContext.SetScoped("dd.env", CorrelationIdentifier.Env.ToString()))
```

Comment:
The `CorrelationIdentifier.Env`, `CorrelationIdentifier.Service`, and `CorrelationIdentifier.Version` APIs are not yet shipped in the .NET Tracer but are work-in-progress

Reply:
```suggestion
using (MappedDiagnosticsLogicalContext.SetScoped("dd.env", Tracer.Instance.Settings.Environment))
```